adapters/adsinteractive/adsinteractive.go

```diff
@@ -0,0 +1,78 @@
+package adsinteractive
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/prebid/openrtb/v17/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+// Builder builds a new instance of the Foo adapter for the given bidder with the given config.
```

Comment:
Thank you for following our adapter dev guide. Please replace Foo here with Adsinteractive, or your company casing AdsInteractive?